brushfire-scalding/src/main/scala/com/stripe/brushfire/scalding/Trainer.scala

```diff
@@ -45,7 +45,7 @@ case class Trainer[K: Ordering, V: Ordering, T: Monoid](
     copy(samplerExecution = newExecution)
   }
 
-  def tee[A](fn: ((TypedPipe[Instance[K, V, T]], Sampler[K], Iterable[(Int, Tree[K, V, T])])) => Execution[A]): Trainer[K, V, T] = {
+  def tee(fn: ((TypedPipe[Instance[K, V, T]], Sampler[K], Iterable[(Int, Tree[K, V, T])])) => Execution[_]): Trainer[K, V, T] = {
```

Comment:
Just in the interests of increasing my Scala knowledge: what difference does this make?